Dockerfile

```diff
@@ -6,4 +6,6 @@ WORKDIR $PROJECT_ROOT
 COPY requirements.txt requirements.txt
 RUN pip install -r requirements.txt
 COPY . .
+COPY django-reference-app/django-reference-app/static/* /usr/local/lib/python3.6/site-packages/govuk_template_base/
+COPY django-reference-app/django-reference-app/static/submit-button.html /usr/local/lib/python3.6/site-packages/govuk_forms/templates/govuk_forms/
```

Comment:
Same as above, but I'm also pretty sure HTML shouldn't be in the `static` directory - instead a templates directory should be configured in `settings.py`